README.md

```diff
@@ -159,6 +159,17 @@ This screen can be closed (by tapping the `X` icon) to access other terminal fea
 public class PayRequest {
     private String amount = null;
     private Boolean showTips = null;
+    private PaymentMethod paymentMethod = null;
+}
+
+public class PaymentMethod {
+  private PaymentMethodType paymentMethodType = null;
+}
+
+public enum PaymentMethodType {
+  CARD("CARD"),
+  ALIPAY("ALIPAY"),
+  WECHAT("WECHAT")
```

Comment:
I think the default enum string values are fine here, so no need to add the string values for each enum.
e.g. `CARD,` etc.

Reply:
Ignore, I guess this is because the other enums are based on the previously generated code